functions/src/offers/index.ts

```diff
@@ -55,7 +50,7 @@ export const triggerEventsWhenOfferIsCreated = functions.firestore
   .onCreate((snapshot: DocumentSnapshot, context: EventContext) => {
     return validateOffer(snapshot.data() as IOffer).catch(errors => {
       console.error('Invalid Offer Found: ', errors);
-      return firestore()
+      return db
```

Comment:
This happens all over the code (a mistake I made initially). We should make sure the cloud functions use `admin` db.

Reply:
okay cool.
I was using db whenever I needed to perform an operation on any other document.
Just left these untouched.
I will fix it in other parts of the code as well